src/components/App.js

```diff
@@ -88,6 +89,8 @@ class App extends Component {
     }
 
     render() {
+        const margin = window.location.pathname.endsWith("cbioportal") ? 0 : 26;
```

Comment:
I guess we change the margin to give some air to the cbioportal iframe?

Reply:
added a clarifying comment